wiki/Gameplay/Note_lock/en.md

```diff
@@ -0,0 +1,32 @@
+---
+tags:
+  - notelock
+  - jittering notes
+  - shaking notes
+---
+
+# Note lock
+
+![](img/notelock.gif "Example of note lock: the second note is inaccessible")
+
+**Note lock** is an informal term for a gameplay mechanic of [osu!](/wiki/Game_mode/osu!) which may prevent a player from clearing a hit object. It happens if **two** conditions are met at the same time:
+
+1. The [timing windows](/wiki/Beatmapping/Overall_difficulty#timing) of two hit objects overlap.
+2. The first object of those two has not been judged yet (hit or missed).
+
+In this case, the second object is said to be *locked* behind the first one, which makes osu! ignore the player's input on it until the first object's hit window has passed. Note lock only becomes a problem if the player is unable to return to the previous object and interact with it properly, thus removing the lock. In this case, a locked note may cause a cascading failure, forcing the player to miss more and more subsequent objects, until they run out of [health](/wiki/Beatmapping/Health) and fail the beatmap.
```

Comment:
```suggestion
In this case, the second object is said to be *locked* behind the first one, which makes osu! ignore the player's input on it until the first object's hit window has passed. Note lock only becomes a problem if the player is unable to return to the previous object and interact with it properly, thus removing the lock. In this case, a locked note may cause a cascading failure, forcing the player to miss more and more subsequent objects, until they run out of [health](/wiki/Beatmapping/Health), and fail the beatmap.
```

Reply:
this is a regular sequential part of the sentence, no serial comma required IMO